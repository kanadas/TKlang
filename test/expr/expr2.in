//Å‚olololo
2 + (-15) * 3
