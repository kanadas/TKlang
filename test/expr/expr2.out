expr/expr2.in
-43
